Documentation/teaching/so2/assign0-kernel-api.rst

```diff
@@ -0,0 +1,120 @@
+=========================
+Assignment 0 - Kernel API
+=========================
+
+-  Deadline: :command:`Wednesday, 24 March 2020, 23:59`
+
+Assignment's Objectives
+=======================
+
+*  getting familiar with the qemu setup
+*  inserting / deleting kernel modules
+*  getting familiar with the list API implemented in the kernel
+*  have fun :)
+
+Statement
+==============
+
+Write a kernel module called `list` (the resulting file must be called `list.ko`) which stores data (strings)
+in an internal list.
+
+It is mandatory to use `the list API <https://github.com/torvalds/linux/blob/master/include/linux/list.h>`__
+implemented in the kernel.
+For details you can take a look `the laboratory 3 <https://ocw.cs.pub.ro/courses/so2/laboratoare/lab03#liste>`__
```

Comment:
typo: "take a look at" or "check laboratory 3"

Reply:
Actually this year it's lab 2 and the link is: https://linux-kernel-labs.github.io/refs/heads/master/so2/lab2-kernel-api.html

Reply:
Fixed